ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -205,13 +206,18 @@ struct ZipBuilder {
     }
 
     let zipDir = try assembleDistributions(inProjectDir: projectDir, withPackageKind: "Firebase",
```

Comment:
Hmm should `withPackageKind:` also be wrapped?

Reply:
Seems like it, but I can't find the right `swiftformat`.  It's stable after manually updating.